infra/feast-operator/internal/controller/services/services.go

```diff
@@ -357,6 +357,10 @@ func (feast *FeastServices) setDeployment(deploy *appsv1.Deployment, feastType F
 func (feast *FeastServices) getContainerCommand(feastType FeastServiceType) []string {
 	deploySettings := FeastServiceConstants[feastType]
 	targetPort := deploySettings.TargetHttpPort
+	logLevel := feast.getLogLevelForType(feastType)
+	if logLevel != nil {
+		deploySettings.Command = append(deploySettings.Command, "--log-level="+strings.ToUpper(*logLevel))
```

Comment:
another nit, but maybe let's follow the same format as the other command additions?
```suggestion
		deploySettings.Command = append(deploySettings.Command, []string{"--log-level", strings.ToUpper(*logLevel)}...)
```